app/services/optout/OptOutService.scala

```diff
@@ -113,6 +116,18 @@ class OptOutService @Inject()(itsaStatusUpdateConnector: ITSAStatusUpdateConnect
   def makeOptOutUpdateRequest(optOutProposition: OptOutProposition, intent: OptOutTaxYear)(implicit user: MtdItUser[_],
                                                                                            hc: HeaderCarrier, ec: ExecutionContext): Future[OptOutUpdateResponse] = {
 
+    def combineByReturningAnyFailureFirstOrAnySuccess(responses: Seq[Future[OptOutUpdateResponse]]): Future[OptOutUpdateResponse] = {
```

Comment:
again, this is bad practice to deal with Future input parameter, if this can be addressed